extractor/extractor/filters.py

```diff
@@ -36,14 +37,14 @@ def filter_index_record(
 ) -> FilterCallback[IndexRecord]:
     def _filter(record: IndexRecord) -> bool:
         passed = True
-        for fieldName in filters:
-            filterText = filters[fieldName]
-            if not hasattr(record, fieldName):
-                raise FieldNotFound(fieldName)
-            value = getattr(record, fieldName)
+        for field_name in filters:
+            filter_text = filters[field_name]
```

Comment:
just curious, did you update all of the camel case to snake case manually or does python have analysis for that or something? ALSO wtf is snake case called snake case because of Python?

Reply:
I think the name comes from the fact that it kinda looks like a_snake_or_whatever...
![image](https://user-images.githubusercontent.com/165338/73380841-f3663e80-4281-11ea-9fe8-5cbef8c9e9e2.png)

The linter complains, so I changed them in PyCharm. In the past I've used lower camel case for local variables so they don't conflict with the snake case public API names, but whatevs.